MailManager/pom.xml

```diff
@@ -63,15 +63,21 @@
     </dependency>
 
     <dependency>
-      <groupId>com.hubspot.dropwizard</groupId>
-      <artifactId>dropwizard-guice</artifactId>
-      <version>${dropwizard-guice.version}</version>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
+      <version>3.0</version>

```

Comment:
I'd prefer to follow our idea of keeping dependencies' versions separately from theirs declarations.